src/ducks/apps/components/ApplicationPage/Details.jsx

```diff
@@ -180,14 +188,16 @@ export const Details = ({ app, description, changes, parent, mobileApps }) => {
             </div>
           </div>
         )}
-        <div>
-          <Button
-            label={t('app_page.permissions.button.label')}
-            className="sto-app-permissions-button"
-            onClick={() => onShowPermissions()}
-            subtle
-          />
-        </div>
+        {!intentData && (
```

Comment:
can you explain in the commit or in a comment, why do we want to hide that?

Reply:
J'ajoute un commentaire et un test 👍

Reply:
Something that explain WHY we don't need to display the content. The if speaks for itself, but you don't give us any context

Reply:
Après échange avec le PO, plus de raison de masquer le pour ouvrir la modale des permissions, ce code n'est plus 👍